src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -12,23 +12,25 @@
 import seedu.address.model.person.Person;
 
 /**
- * Shows the full view for a person with all of their information.
+ * Toggles the card status of a person to either show all of their information,
+ * or only an abridged version of it.
  */
 public class ViewCommand extends Command {
 
     public static final String COMMAND_WORD = "view";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Shows all information about the person identified by the index number in the displayed person list.\n"
+            + ": Toggles the amount of information shown about the person "
+            + "identified by the index number in the displayed person list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_VIEW_SUCCESS = "Showing information for %1$s.";
+    public static final String MESSAGE_VIEW_SUCCESS = "Toggling the view of %1$s.";
```

Comment:
That's a great suggestion and it makes a lot of sense to me! I can't think of a good way to implement this at the moment, but I'll create an issue so that we can keep this in mind and improve the view command with this feature in the future!